community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/tracing/LogAppendEvent.java

```diff
@@ -25,8 +25,7 @@
  */
 public interface LogAppendEvent extends LogForceEvents, AutoCloseable
 {
-
-    LogAppendEvent NULL = new LogAppendEvent()
+    class Null implements LogAppendEvent
```

Comment:
Why do we need to introduce this `Null` class?
All event have ''concept'' of null event instance for example https://github.com/neo4j/neo4j/blob/2.3/community/kernel/src/main/java/org/neo4j/kernel/impl/transaction/tracing/CommitEvent.java#L27. Is there is any reason why this should be different? If so can we please rename it to something else (Empty for example)?